CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Contributing
+
+Please remember that copying and pasting mappings from alternate projects under more restrictive licenses (such as MCP or Mojang's obfuscation maps)
```

Comment:
does the rule of Mojang's obfuscation maps really apply to versions where they cant be accessed?

Reply:
idk, it's just here as an example anyway
~_perhaps they will release them someday_~